### ISSUE TITLE
Hebrew words are cut in the bottom of last line

### ISSUE THREAD

**Issue by thecodeside** [AndroidDeveloperLB/AutoFitTextView#52]
In my app I have sentence with full screen width on a welcome screen.
When text is in a Hebrew language the words are cut in the bottom of last line.

Adding bottom padding or setting clipToPadding/clipChildren don't help. 

Can't use the Google autosizing TextView because it still has a "breaking the words issue".

**Comment by AndroidDeveloperLB** [comment]
Yes, I know it has issues, and that Google has even worse issues.
Sadly I'm not familiar anymore with the code I've put here.
If you know how to fix it, please show a sample that shows the issue, and that indeed you solved it.
Also, please consider writing about auto-resizing of font size of Google in its own issue tracker:
https://issuetracker.google.com/issues

BTW, I also know Hebrew (my main language), and I know it's even more problematic :(
